engines/kyra/gui/gui_v1.cpp

```diff
@@ -599,26 +599,34 @@ void MainMenu::printString(const char *format, int x, int y, int col1, int col2,
 	Common::String string = Common::String::vformat(format, vaList);
 	va_end(vaList);
 
+	Common::String revBuffer;
+	const char *cstr = string.c_str();
+	if (_vm->gameFlags().lang == Common::HE_ISR) {
+		for (int i = string.size() - 1; i >= 0; --i)
+			revBuffer += string[i];
```

Comment:
Why don't use you `Common::convertBiDiString`?

Reply:
I have tried, but IIRC it had issues with brackets (like in "[ Autosave ]"), so I just opted for reversing the texts, knowing the result will always be the exact same text spelled backwards is very predictable.